.github/workflows/convention-develocity-gradle-plugin-verification.yml

```diff
@@ -69,6 +69,9 @@ jobs:
           - sample: '6.9_and_later'
             version: 'wrapper'
             java: '21'
+          - sample: '6.9_and_later'
+            version: 'release-candidate'
```

Comment:
I like the idea of testing RC versions, but I don't know if this is the right way to do it. A better way would be to have a nightly job that runs against the RC version. As it is now, RCs will only be tested if an unrelated PR is opened. For example, Dependabot creates a PR to upgrade a dependency, and the build fails because of some unrelated incompatibility with a Gradle RC.

Reply:
Yes, that's true. Do you think we should simply add a cron trigger to the verification workflows, or do you propose creating a new one to just run against the RC. Bear in mind that we'd be doing this only for Gradle.

Reply:
Deferred to a later PR (after a discussion).